wranglertools/import_data.py

```diff
@@ -150,65 +171,65 @@ def md5(path_string):
 
 class WebFetchException(Exception):
     """
-    custom exception to raise if ftp or http fetch fails
+    custom exception to raise if http fetch fails
     """
     pass
 
 
+def google_authenticate():
+    gsauth = None
+    creddir = pp.Path(__file__).parent.joinpath('.config', 'gspread')
+    gsauth = gspread.oauth(
+        credentials_filename=creddir.joinpath('credentials.json'),
+        authorized_user_filename=creddir.joinpath('authorized_user.json'),
+        scopes=SCOPES
+    )
+    return gsauth
```

Comment:
Will this work for users who `pip install` the package? You probably want to allow folks to set a path in an environment variable as well and maybe fallback to this if not found? I would also definitely add everything in `.config` to `.gitignore`.